CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_FLAGS "-DROOT_DIR=\\\"${CMAKE_SOURCE_DIR}\\\" -DVERSION=\\\"${PROJECT_VERSION}\\\" ")
 
 # Enable or Disable Clang Tidy in Build
-option(CLANG_TIDY "Enable Clang Tidy in Build" OFF)
+option(CLANG_TIDY "Enable Clang Tidy in Build" ON)
```

Comment:
I think that this can be left OFF by default. We usually only switch it ON as part of the IC. If it's ON by default, people are going to get a huge dump of warnings every time they build on their machines.